### ISSUE TITLE
Get the luminance of a color

### ISSUE THREAD

**Issue by yannickl** [yannickl/DynamicColor#23]
Get the relative luminosity value of a color following the WCAG specs: https://www.w3.org/TR/WCAG20/#relativeluminancedef

**Comment by Onejjy** [comment]
I'm on it ;)

**Comment by yannickl** [comment]
I'm thinking of something. Is the `luminance` method should be a property instead? It'll be more Swift friendly no?

**Comment by Onejjy** [comment]
Indeed, thats more swifty :)
